src/layout/Footer.tsx

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import { FaFacebook, FaInstagram, FaTwitter, FaLinkedin } from 'react-icons/fa';
+
+// interface for social media links
+interface SocialLink {
```

Comment:
You can remove this interface as it's a static component, so we don't really need it